src/components/Button/index.tsx

```diff
@@ -374,6 +375,7 @@ const FwdButton = ({
   to,
   tooltip,
   tooltipBaseId,
+  ariaLabel,
```

Comment:
aria-label is only required when there is no explicit text content, ie only a few of our buttons (icon buttons)

Reply:
It's more clear, thanks for feedback